app/javascript/react/components/UserTile.js

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+const UserTile = (props) => {
+  
+  return (
+    <h5><strong>{props.username}</strong></h5>
+  )
+}
+
+export default UserTile
```

Comment:
we'll delete this in favor of moving this logic back into the ReviewTile 

also: do not use the <strong> tag. instead use CSS to increase font weight if you want that